Platforms/QemuQ35Pkg/QemuQ35Pkg.fdf

```diff
@@ -274,6 +274,8 @@ INF  PolicyServicePkg/PolicyService/Pei/PolicyPei.inf
 INF  QemuQ35Pkg/ConfigKnobs/ConfigKnobs.inf
 INF  OemPkg/OemConfigPolicyCreatorPei/OemConfigPolicyCreatorPei.inf
 
+INF  DebuggerFeaturePkg/DebugConfigPei/DebugConfigPei.inf

```

Comment:
Should this and the libs be wrapped in if DEBUGGER_ENABLED to only build/include if we are going to be using the debugger?

Reply:
It could be, but the general design of this module is to leverage the device state like we would on a real system so I think it's best to leave it in and rely on the device state check on whether the debugger actually get enabled.